packages/eslint-config/rules/style.js

```diff
@@ -0,0 +1,775 @@
+    'brace-style': ['error', '1tbs', {
+      allowSingleLine: false,
+    }],
+
+
+    /**
+     * enforce camelcase naming convention
+     */
+    'camelcase': ['error', {
+      properties: 'always',
+      ignoreDestructuring: false,
+      allow: [
+        '^UNSAFE_', // for unsafe React lifecycle methods
+      ],
+    }],
+
+
+    /**
+     * enforce or disallow capitalization of the first letter of a comment
+     */
+    'capitalized-comments': ['off'],
+
+
+    /**
+     * require or disallow trailing commas
+     */
+    'comma-dangle': ['error', 'always-multiline'],
+
+
+    /**
+     * enforce consistent spacing before and after commas
+     */
+    'comma-spacing': ['error', {
+      before: false,
+      after: true,
+    }],
+
+
+    /**
+     * enforce consistent comma style
+     */
+    'comma-style': ['error', 'last', {
+      exceptions: { // NO EXCEPTIONS
+        ArrayExpression: false,
+        ArrayPattern: false,
+        ArrowFunctionExpression: false,
+        CallExpression: false,
+        FunctionDeclaration: false,
+        FunctionExpression: false,
+        ImportDeclaration: false,
+        ObjectExpression: false,
+        ObjectPattern: false,
+        VariableDeclaration: false,
+        NewExpression: false,
+      },
+    }],
+
+
+    /**
+     * enforce consistent spacing inside computed property brackets
+     */
+    'computed-property-spacing': ['error', 'never', { enforceForClassMembers: true }],
+
+
+    /**
+     * enforce consistent naming when capturing the current execution context
+     */
+    'consistent-this': ['error', 'self'],
+
+
+    /**
+     * require or disallow newline at the end of files
+     */
+    'eol-last': ['error', 'always'],
+
+
+    /**
+     * require or disallow spacing between function identifiers and their invocations
+     */
+    'func-call-spacing': ['error', 'never'],
+
+
+    /**
+     * require function names to match the name of the variable or property to which they are assigned
+     */
+    'func-name-matching': ['error', 'always', {
+      considerPropertyDescriptor: true,
+      includeCommonJSModuleExports: true,
+    }],
+
+
+    /**
+     * require or disallow named function expressions
+     */
+    'func-names': ['error', 'always', {
+      generators: 'always',
+    }],
+
+
+    /**
+     * enforce the consistent use of either function declarations or expressions
+     */
+    'func-style': ['error', 'declaration', {
+      allowArrowFunctions: true,
+    }],
+
+
+    /**
+     * enforce line breaks between arguments of a function call
+     */
+    'function-call-argument-newline': ['error', 'consistent'],
+
+
+    /**
+     * enforce consistent line breaks inside function parentheses
+     */
+    'function-paren-newline': ['error', 'consistent'],
+
+
+    /**
+     * disallow specified identifiers
+     */
+    'id-blacklist': ['off'],
+
+
+    /**
+     * enforce minimum and maximum identifier lengths
+     */
+    'id-length': ['error', {
```

Comment:
Could you elaborate on this one? I'm not sure what you mean

Reply:
for example

```js
         |> Unused arg 1
         |  |> Unused arg 2 
doStuff (_, __, usedArg) {
}
```

I have a number of functions where I only need the second argument to complete it's task, but because it is a part of a set of functions with standardized arguments, it makes no sense to accept the second argument as the first. Sometimes these functions can be complex so I'd like to make it clear that first argument is ignored.

Reply:
Okay with me